greenpithumb/db_store.py

```diff
@@ -2,6 +2,18 @@
 from dateutil import parser
 
 
+def _serialize_timestamp(timestamp):
+    """Converts a timestamp to a string.
+
+    Args:
+        timestamp: Timestamp in datetime.datetime format.
+
+    Returns:
+        Timestamp as a string in ISO 8601 format.
+    """
+    return timestamp.isoformat(' ')
```

Comment:
Suggest using `'T'` a the separator character because that's most common for ISO 8601 strings.